ocaml/idl/datamodel.ml

```diff
@@ -4780,6 +4780,11 @@ module VGPU = struct
         field ~qualifier:DynamicRO ~ty:(Ref _pgpu) ~lifecycle:[Published, rel_dundee, ""] "scheduled_to_be_resident_on" "The PGPU on which this VGPU is scheduled to run" ~default_value:(Some (VRef null_ref));
         field ~qualifier:DynamicRO ~ty:(Map (String,String)) ~lifecycle:[Published, rel_inverness, ""] ~default_value:(Some (VMap [])) "compatibility_metadata" "VGPU metadata to determine whether a VGPU can migrate between two PGPUs";
         field ~qualifier:RW ~ty:String ~lifecycle:[Published, rel_quebec, ""] ~default_value:(Some (VString "")) "extra_args" "Extra arguments for vGPU and passed to demu";
+        field ~qualifier:DynamicRO
+          ~ty:(Ref _pci)
+          ~lifecycle:[Published, rel_quebec, ""]
+          ~default_value:(Some (VRef null_ref))
+          "pci" "PCI device";
```

Comment:
For consistency with other API fields, we should capitalise this name: `PCI`.
Also, please explain in the doc string that this is the PCI device that is passed through to the VM for this VGPU, either a full device pass-through, or an SR-IOV virtual function.